doc/api/process.md

```diff
@@ -4059,6 +4059,16 @@ This property refers to the value of underlying file descriptor of
 `process.stderr`. The value is fixed at `2`. In [`Worker`][] threads,
 this field does not exist.
 
+## `process.setTraceSigInt(enable)`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* `enable` {boolean}
+
+Enable or disable printing a stack trace on `SIGINT`. The API is only available on the main thread.
```

Comment:
Just as a general rule I'm fairly reluctant to add new APIs to `process`. Could this be an environment variable instead?

Reply:
Done !

Reply:
I feel like signal related APIs are already on `process`. So it might not hurt to have another signal API on `process`... But I don't feel strongly about this.